srcpkgs/helvum/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'helvum'
+pkgname=helvum
+version=0.3.4
+revision=1
+build_style=cargo
+build_helper=rust
+hostmakedepends="pkg-config clang libclang"
+makedepends="gtk4-devel pipewire-devel glib-devel"
+depends="gtk4 pipewire"
+short_desc="GTK patchbay for pipewire"
+maintainer="classabbyamp <dev@kb6.ee>"
+license="GPL-3.0-only"
+homepage="https://gitlab.freedesktop.org/pipewire/helvum"
+distfiles="https://gitlab.freedesktop.org/pipewire/helvum/-/archive/${version}/helvum-${version}.tar.gz
+ https://gitlab.freedesktop.org/pipewire/pipewire-rs/-/archive/v0.4.1/pipewire-rs-v0.4.1.tar.gz"
```

Comment:
could you create a variable _pipewire_rs_version, put it next to $version and use it here?

Reply:
done.